app/views/main.html

```diff
@@ -0,0 +1,23 @@
+<h2>Awesome Todo-List</h2>
+<div class="container">
+
+  <form role="form" ng-submit="addTodo()">
+    <div class="row">
+      <div class="input-group">
+        <input type="text1" ng-model="todo" placeholder="What do you want to do?" class="form-control" required>
+        <span class="input-group-btn">
+          <input type="submit" class="btn btn-danger" value="Add">
+        </span>
+      </div>
+    </div>
+  </form>
+  <br>
+  <div ui-sortable ng-model="todos">
+    <p class="input-group" ng-repeat="todo in todos" style="padding:5px 10px; cursor: move;">
+      <input type="text" ng-model="todo" class="form-control">
+        <span class="input-group-btn">
+          <button class="btn btn-seb" ng-click="removeTodo($index)" aria-label="Remove">X</button>
```

Comment:
_Always_ prefer passing objects rather than arbitrary identifiers.  You are relying here on the `$index` in your view being the same as the index position of the object in the source collection.  This is very dangerous - what happens if your view is ordered differently?